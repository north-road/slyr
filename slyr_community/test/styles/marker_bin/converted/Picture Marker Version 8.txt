[{'class': 'QgsSvgMarkerSymbolLayer',
  'name': '<svg xmlns="http://www.w3.org/2000/svg" '
          'xmlns:svg="http://www.w3.org/2000/svg" id="svg2" version="1.0">\n'
          ' <defs id="defs1">\n'
          '  <pattern height="6" id="EMFhbasepattern" '
          'patternUnits="userSpaceOnUse" width="6" x="0" y="0"></pattern>\n'
          ' </defs>\n'
          ' <path d="m 12.333334,21 c -0.333334,-1 -1,-1 -1.666667,-1.333334 '
          '-0.6666673,0 -0.6666673,0 -0.6666673,0 C 9.6666667,19.333333 '
          '9.6666667,19.333333 9.6666667,19.333333 9.3333334,19 9.3333334,19 '
          '9.3333334,19 8.6666667,18.333333 8.6666667,18.333333 '
          '8.6666667,18.333333 8,18 8,18 8,18 7.6666667,17.666666 '
          '7.6666667,17.666666 7.6666667,17.666666 c 0,0.333334 0,0.666667 '
          '0,1.333334 -1,-1 -0.6666667,-1 -1,-2.333334 0,-0.333333 0,-1.666666 '
          '0,-2.333333 C 7,14.666666 7,14.666666 7,14.666666 7.3333334,15 '
          '7.3333334,15 7.3333334,15 8,15.666666 8,15.666666 8,15.666666 c '
          '0.6666667,0.666667 0.6666667,0.666667 0.6666667,0.666667 0,0 0,0 '
          '0,0 C 9,16.666666 9,16.666666 9,16.666666 c 0,0 0,0 0,0 0,0 0,0 0,0 '
          '-0.3333333,0 -0.3333333,0 -0.3333333,0 -0.3333333,0 -0.3333333,0 '
          '-0.3333333,0 0,0.333334 0,0.333334 0,0.333334 C 9,17.666666 '
          '11.666667,19.666666 11.666667,17.333333 c 0,-1.333333 0,-2.333333 '
          '0,-3.666667 C 12,13 12,13 12,13 c 0,-1.333334 0,-1.333334 '
          '0,-1.333334 0,-1 0,-1 0,-1 C 12,10 12,10 12,10 12,9 12,9 12,9 12,9 '
          '12,9 12,9 8,8.6666666 8,8.6666666 8,8.6666666 8,8 8,8 8,8 c 4,0 4,0 '
          '4,0 0,-0.6666667 0,-0.6666667 0,-0.6666667 0,0 0,0 0,0 C '
          '11.666667,7 11.666667,7 11.666667,7 c 0,0 0,0 0,0 '
          '-0.333333,-0.3333334 -0.333333,-0.3333334 -0.333333,-0.3333334 0,0 '
          '0,0 0,0 C 11,6 11,6 11,6 c 0,0 0,0 0,0 0,-1 0.666667,-1.3333334 '
          '1.333334,-1.6666667 0,1 0,1 0,1 0,0 0,0 0,0 0,0 0,0 0,0 -0.333334,0 '
          '-0.333334,0 -0.333334,0 0,0.3333333 0,0.3333333 0,0.3333333 '
          '-0.333333,0 -0.333333,0 -0.333333,0 0,0 0,0 0,0 0,0.3333334 '
          '0,0.3333334 0,0.3333334 0,0 0,0 0,0 C 12,6.3333333 12,6.3333333 '
          '12,6.3333333 12,6.6666666 12,6.6666666 12,6.6666666 '
          '12.333334,6.3333333 12,6 12.333334,6 12.666667,6 12.666667,6 '
          '12.666667,6 13,6 13,6 13,6 c 0,0.6666666 0,0.6666666 0,0.6666666 '
          '0,-0.3333333 0,-0.3333333 0,-0.3333333 0,0 0,0 0,0 C 13.333334,6 '
          '13.333334,6 13.333334,6 c 0,-0.3333334 0,-0.3333334 0,-0.3333334 '
          '-0.333334,0 -0.333334,0 -0.333334,0 0,0 0,0 0,0 0,-0.3333333 '
          '0,-0.3333333 0,-0.3333333 -0.333333,0 -0.333333,0 -0.333333,0 0,0 '
          '0,0 0,0 -0.333333,0 -0.333333,0 -0.333333,0 0,-1 0,-1 0,-1 C '
          '13.666667,4.6666666 13.666667,4.6666666 14,6 c -0.333333,0 '
          '-0.333333,0 -0.333333,0 0,0.3333333 0,0.3333333 0,0.3333333 '
          '0,0.3333333 0,0.3333333 0,0.3333333 0,0 0,0 0,0 C 13.333334,7 '
          '13.333334,7 13.333334,7 c 0,0 0,0 0,0 C 13,7.3333333 13,7.3333333 '
          '13,7.3333333 c -0.333333,0 -0.333333,0 -0.333333,0 C 13,8 13,8 13,8 '
          'c 4,0 4,0 4,0 0,0.6666666 0,0.6666666 0,0.6666666 C 13,9 13,9 13,9 '
          'c 0,2.666666 0.333334,5.666666 0.333334,8.333333 0,0.333333 '
          '0,1.333333 1,1 0,0 0,0 0,0 0.333333,0 0.333333,0 0.333333,0 C 15,18 '
          '15,18 15,18 c 0.333334,0 0.333334,0 0.333334,0 0.333333,-0.333334 '
          '0.333333,-0.333334 0.333333,-0.333334 0.666667,-0.333333 '
          '0.666667,-0.333333 0.666667,-0.333333 C 16.666667,17 16.666667,17 '
          '16.666667,17 c 0,0 0,0 0,0 0,-0.333334 0,-0.333334 0,-0.333334 '
          '-0.333333,0 -0.333333,0 -0.333333,0 -0.333334,0 -0.333334,0 '
          '-0.333334,0 0,0 0,0 0,0 1,-0.666666 1.333334,-1 2.333334,-2.333333 '
          '0,1.666667 0.333333,3.666667 -1,4.666667 0,-0.666667 0,-0.666667 '
          '0,-0.666667 0,-0.333333 0,-0.333333 0,-0.333333 0,-0.333334 '
          '0,-0.333334 0,-0.333334 0,0 0,0 0,0 -1,0.666667 -1.333334,1.333334 '
          '-2.333334,2 -0.666666,0 -0.666666,0 -0.666666,0 C 14,20 14,20 14,20 '
          'c -0.333333,0 -0.333333,0 -0.333333,0 0,0 0,0 0,0 C 13,20.333333 '
          '13,20.333333 13,20.333333 c -0.333333,0.333333 -0.333333,0.333333 '
          '-0.333333,0.333333 z" id="path1" '
          'style="fill:none;stroke:#000000;stroke-width:0.666667px;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"></path>\n'
          ' <path d="M 0.33333334,25.666666 V 0.33333334 H 24.666667 V '
          '25.666666 Z" id="path2" '
          'style="fill:none;stroke:#000000;stroke-width:0.666667px;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"></path>\n'
          '</svg>',
  'offset_unit': 'Point',
  'size': '15',
  'size_unit': 'Point'}]
