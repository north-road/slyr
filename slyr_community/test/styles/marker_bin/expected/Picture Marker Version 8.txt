{'halo': False,
 'halo_size': 2.0,
 'halo_symbol': {'color': {'B': 255,
                           'G': 255,
                           'R': 255,
                           'dither': True,
                           'is_null': False,
                           'stream_offset': '0xdc',
                           'type': 'RgbColor',
                           'version': 1},
                 'enabled': True,
                 'fill_style': 'solid',
                 'locked': False,
                 'outline': {'color': {'B': 110,
                                       'G': 110,
                                       'R': 110,
                                       'dither': True,
                                       'is_null': False,
                                       'stream_offset': '0x99',
                                       'type': 'RgbColor',
                                       'version': 1},
                             'enabled': True,
                             'line_type': 'solid',
                             'locked': False,
                             'stream_offset': '0x87',
                             'symbol_level': 0,
                             'type': 'SimpleLineSymbol',
                             'version': 1,
                             'width': 0.0},
                 'stream_offset': '0x75',
                 'symbol_level': 0,
                 'type': 'SimpleFillSymbol',
                 'version': 1},
 'levels': [{'angle': 0.0,
             'color_background': {'B': 255,
                                  'G': 255,
                                  'R': 255,
                                  'dither': True,
                                  'is_null': False,
                                  'stream_offset': '0x882',
                                  'type': 'RgbColor',
                                  'version': 1},
             'color_foreground': {'B': 0,
                                  'G': 0,
                                  'R': 0,
                                  'dither': False,
                                  'is_null': True,
                                  'stream_offset': '0x853',
                                  'type': 'RgbColor',
                                  'version': 1},
             'color_transparent': None,
             'enabled': True,
             'locked': False,
             'picture': {'content': b'AQAAAHQAAACAAQAAwv3//5kBAADc/f//4C4AAO65'
                                    b'///tMQAAG73//yBFTUYAAAEA+AYAABUAAAADAAAA'
                                    b'DQAAAFgAAAAAAAAAAAQAAAADAABAAQAA8AAAAEEA'
                                    b'ZABvAGIAZQAgAFMAeQBzAHQAZQBtAHMAAAARAAAA'
                                    b'DAAAAAgAAAASAAAADAAAAAEAAAAKAAAAEAAAAAAA'
                                    b'AAAAAAAADAAAABAAAAAAAAAAAAAAAAkAAAAQAAAA'
                                    b'4QAAAOoAAAALAAAAEAAAAEsAAABOAAAAJwAAABgA'
                                    b'AAABAAAAAQAAAAAAAAAAAAAAJQAAAAwAAAABAAAA'
                                    b'XwAAADQAAAACAAAALAAAAAAAAAAsAAAAAAAAAAAA'
                                    b'AQACAAAAAAAAAAAAAAAAAAAAAAAAACUAAAAMAAAA'
                                    b'AgAAADsAAAAIAAAAGwAAABAAAACmBAAAhPn//1gA'
                                    b'AAAgBQAAhgEAAMX9//+UAQAA1/3//0EBAAClBIH5'
                                    b'owSB+aEEgPmfBID5nwSA+Z8EgPmeBH/5ngR/+Z4E'
                                    b'f/mdBH75nQR++Z0EfvmbBHz5mwR8+ZsEfPmZBHv5'
                                    b'mQR7+ZkEe/mYBHr5mAR6+ZgEevmYBHv5mAR8+ZgE'
                                    b'fvmVBHv5lgR7+ZUEd/mVBHb5lQRy+ZUEcPmWBHH5'
                                    b'lgRx+ZYEcfmXBHL5lwRy+ZcEcvmZBHT5mQR0+ZkE'
                                    b'dPmbBHb5mwR2+ZsEdvmbBHb5mwR2+ZsEdvmcBHf5'
                                    b'nAR3+ZwEd/mcBHf5nAR3+ZwEd/mcBHf5nAR3+ZwE'
                                    b'd/mbBHf5mwR3+ZsEd/maBHf5mgR3+ZoEd/maBHj5'
                                    b'mgR4+ZoEePmcBHr5pASA+aQEefmkBHX5pARy+aQE'
                                    b'bvmlBGz5pQRs+aUEbPmlBGj5pQRo+aUEaPmlBGX5'
                                    b'pQRl+aUEZfmlBGP5pQRj+aUEY/mlBGD5pQRg+aUE'
                                    b'YPmlBGD5pQRg+aUEYPmZBF/5mQRf+ZkEX/mZBF35'
                                    b'mQRd+ZkEXfmlBF35pQRd+aUEXfmlBFv5pQRb+aUE'
                                    b'W/mlBFv5pQRb+aUEW/mkBFr5pARa+aQEWvmkBFr5'
                                    b'pARa+aQEWvmjBFn5owRZ+aMEWfmjBFn5owRZ+aME'
                                    b'WfmiBFf5ogRX+aIEV/miBFf5ogRX+aIEV/miBFT5'
                                    b'pART+aYEUvmmBFX5pgRV+aYEVfmmBFX5pgRV+aYE'
                                    b'VfmmBFX5pgRV+aYEVfmlBFX5pQRV+aUEVfmlBFb5'
                                    b'pQRW+aUEVvmkBFb5pARW+aQEVvmkBFb5pARW+aQE'
                                    b'VvmkBFf5pARX+aQEV/mkBFf5pARX+aQEV/mlBFj5'
                                    b'pQRY+aUEWPmlBFn5pQRZ+aUEWfmmBFj5pQRX+aYE'
                                    b'V/mnBFf5pwRX+acEV/moBFf5qARX+agEV/moBFn5'
                                    b'qARZ+agEWfmoBFj5qARY+agEWPmoBFj5qARY+agE'
                                    b'WPmpBFf5qQRX+akEV/mpBFb5qQRW+akEVvmoBFb5'
                                    b'qARW+agEVvmoBFb5qARW+agEVvmoBFX5qARV+agE'
                                    b'VfmnBFX5pwRV+acEVfmnBFX5pwRV+acEVfmmBFX5'
                                    b'pgRV+aYEVfmmBFL5pgRS+aYEUvmqBFP5qgRT+asE'
                                    b'V/mqBFf5qgRX+aoEV/mqBFj5qgRY+aoEWPmqBFn5'
                                    b'qgRZ+aoEWfmqBFn5qgRZ+aoEWfmpBFr5qQRa+akE'
                                    b'WvmpBFr5qQRa+akEWvmoBFv5qARb+agEW/mnBFv5'
                                    b'pwRb+acEW/moBF35qARd+agEXfm0BF35tARd+bQE'
                                    b'Xfm0BF/5tARf+bQEX/moBGD5qARg+agEYPmoBGj5'
                                    b'qQRx+akEefmpBHr5qQR9+awEfPmsBHz5rAR8+awE'
                                    b'fPmtBHz5rQR8+a0EfPmuBHv5rgR7+a4Ee/mvBHv5'
                                    b'rwR7+a8Ee/mwBHr5sAR6+bAEevmyBHn5sgR5+bIE'
                                    b'efmzBHj5swR4+bMEePmzBHj5swR4+bMEePmzBHf5'
                                    b'swR3+bMEd/myBHf5sgR3+bIEd/mxBHf5sQR3+bEE'
                                    b'd/mxBHf5sQR3+bEEd/m0BHX5tQR0+bgEcPm4BHX5'
                                    b'uQR7+bUEfvm1BHz5tQR8+bUEfPm1BHv5tQR7+bUE'
                                    b'e/m1BHr5tQR6+bUEevm1BHr5tQR6+bUEevmyBHz5'
                                    b'sQR++a4EgPmsBID5rASA+awEgPmrBIH5qwSB+asE'
                                    b'gfmqBIH5qgSB+aoEgfmqBIH5qgSB+aoEgfmoBIL5'
                                    b'qASC+agEgvmnBIP5pwSD+acEg/k9AAAACAAAADwA'
                                    b'AAAIAAAAQAAAABgAAACfhgEAn4YBAGF5/v9hef7/'
                                    b'WwAAADwAAACAAQAAwf3//5oBAADc/f//AQAAAAYA'
                                    b'AAAGAAAAggSS+YIERvnLBEb5ywSS+YIEkvmCBJL5'
                                    b'KAAAAAwAAAABAAAAKAAAAAwAAAACAAAADgAAABQA'
                                    b'AAAAAAAAEAAAABQAAAA=',
                         'picture_type': 'EmfPicture',
                         'stream_offset': '0x133',
                         'type': 'StdPicture',
                         'version': 1},
             'rotate_with_transform': True,
             'size': 15.0,
             'stream_offset': '0x11b',
             'swap_fg_bg': False,
             'symbol_level': 0,
             'type': 'PictureMarkerSymbol',
             'version': 8,
             'x_offset': 0.0,
             'x_scale': 1.0,
             'y_offset': 0.0,
             'y_scale': 1.0}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerMarkerSymbol',
 'version': 3}
